onnxruntime/contrib_ops/cuda/bert/longformer_attention_impl.cu

```diff
@@ -368,7 +368,7 @@ Status LaunchLongformerSoftmaxKernel(
     const void* q,                // transposed Q with shape (B, N, S, H)
     const void* k,                // transposed K with shape (B, N, S, H)
     const void* v,                // transposed V with shape (B, N, S, H)
-    const void* attention_mask,   // attention mask with shape (B, S), with value 0 not masked and -10000 masked.
+    const void* attention_mask,   // attention mask with shape (B, S), with value 0 not masked and -10000 or torch.finfo(dtype).min masked.
```

Comment:
It might be better to change "torch.finfo(dtype).min" to value of mask_filter_value attribute